services/fluxmonitor_test.go

```diff
@@ -21,11 +19,7 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
-// does not print big.int pointers
-func prettyPrint(i interface{}) string {
-	s, _ := json.MarshalIndent(i, "", "\t")
-	return string(s)
-}
+const COMMON_JOB_TRIGGER_TIMEOUT = 200 * time.Millisecond
```

Comment:
Surprised CI didn't catch this, but we should use camelcase instead of snake case in Go :sweat_smile: 

```suggestion
const CommonJobTriggerTimeout = 200 * time.Millisecond
```